peer-test-suite/docs/insecure_config.yaml

```
@@ -1,5 +1,11 @@
-admin:
-  password: "P@ssw0rd"  # secret in YAML
-database:
-  host: "localhost"
-  port: 5432
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)